src/databaseService/index.js

```diff
@@ -108,7 +108,23 @@ export const getAnsweredDiagnosesForCurrentUser = async (howMany = 25, diagnoseP
       .limit(howMany)
       .get()
 
-    return querySnapshot.docs.map(doc => ({ ...doc.data() }))
```

Comment:
I do not see this function being used in the pull, It could still we working after this change, but it seems more like a refactoring than a functionality. Be careful when doing these things, even if something worked for a similar case, you could still be introducing bugs. As an instance, doc.data() returns a DocumentData, which may or may not be as accessible as a plain object.

Reply:
The reason I changed it... https://github.com/WeDevelop-ARG/cannabis-app/pull/10#discussion_r341626483

Reply:
What you say is very true, Mauro.

I asked him for this change because I knew it'd work: Everything in JavaScript is an object (well, almost everything). `doc.data()` returns an object, whose prototype chain contains `DocumentData`. Any object in JavaScript have _enumerable_ and _non-enumerable_ properties. The object spread syntax (`{ ...obj }`) is just syntactic sugar to copy all _enumerable_ properties from one object to the other. When you use spread, you're actually **not copying** some properties from the original object, the properties that are _non-enumerable_.

In cases like the Firebase Library, they only tell you that the object is an instance of certain class (`DocumentData`), but they don't give information on how the properties are defined on that object. So, to be safe, it's best to avoid copying the properties here. Other libraries (such as Sequelize) contain methods to obtain a plain JavaScript object from their custom one, so that you know what you're working with and get the expected results.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax